spec/controllers/works_controller_spec.rb

```diff
@@ -647,10 +647,10 @@
           work.resource.description = nil
           work.save
           sign_in user
-          post :completed, params: { id: work.id }
+          post :validate, params: { id: work.id }
+          expect(response).to render_template("edit")
           expect(response.status).to be 422
           expect(work.reload).to be_draft
```

Comment:
It would still be good to validate the errors here

Reply:
Good idea. Done.